sr_utilities_common/src/sr_utilities_common/aws_manager.py

```diff
@@ -110,22 +111,59 @@ def _prepare_structure_download(self, files_base_path, files_folder_path, file_n
             else:
                 self.aws_paths.append(f"{file_name}")
 
-    def download(self, bucket_name, files_base_path, files_folder_path, file_names, bucket_subfolder):
+    def download(self, bucket_name, files_base_path, files_folder_path,
+                 file_names, bucket_subfolder, preserve_downloaded_folder_structure=False,
+                 file_extentions_to_exclude=None):
+        if file_extentions_to_exclude is not None:
+            examples = "(e.g. ['.txt', '.jpg'])"
```

Comment:
May be better to call the var `EXAMPLES` as it is a constant

Reply:
not sure our linter would like that

Reply:
fair point